internal/cli/universal_login.go

```diff
@@ -213,14 +215,6 @@ func updateUniversalLoginCmd(cli *cli) *cobra.Command {
 				b.Font = &management.BrandingFont{URL: &inputs.CustomFontURL}
 			}
 
-			if b.Font != nil {
```

Comment:
From testing, this code is not useful. The only case where `b.Font` is not nil is when `inputs.CustomFontURL` is not set due to the if on line 212.

Given that, we will always repeat exactly what the if statement on 212 achieves, i.e. if a `--font` option has been provided then set it